app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    personal_list = [
+        Person(person["name"], person["age"])
+        for person in people
+    ]
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife =\
+                Person.people[person["wife"]]
```

Comment:
Write in single line
```suggestion
            Person.people[person["name"]].wife = Person.people[person["wife"]]
```

Reply:
Or 
wife = Person.people[person["wife"]]
Person.people[person["name"]].wife = wife